### PR TITLE
Depend on latest kube version

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

-->
```release-note

csi-lib-utils library package dependencies are updated to make use of latest kube version v1.21.0 and also v1.4.0 version of csi spec which include many bug fixes and enhancements, with this release. 

```

### CODE REVIEW

**Comment by pohly** [comment]
Please check the test failures.

**Comment by humblec** [comment]
> Please check the test failures.

Sure.. checking on it @pohly , interestingly before sending the PR I checked make and it worked. The PR fails on` make test` , and it seems to be caused by the CSI spec upgrade. Working on it :)

**Comment by humblec** [comment]
@pohly it should be good to go, ptal  :+1: .. Thanks !

**Comment by pohly** [comment]
Please add the changes from your last commit to the commits where they belong (CSI update for the new method, not sure about the go.sum change). Or squash everything - your choice.

**Comment by pohly** [comment]
Please also add a release note entry.

/kind cleanup

**Comment by humblec** [comment]
> Please also add a release note entry.
> 
> /kind cleanup

Squashed commits and also added release note. Please suggest if there any changes required for release note.

File: go.mod [hunk 1]
```diff
@@ -3,61 +3,24 @@ module github.com/kubernetes-csi/csi-lib-utils
...
-replace k8s.io/kubectl => k8s.io/kubectl v0.19.0
-
-replace k8s.io/kubelet => k8s.io/kubelet v0.19.0
-
-replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.0
-
-replace k8s.io/metrics => k8s.io/metrics v0.19.0
+replace k8s.io/component-base => k8s.io/component-base v0.21.1
 
 replace k8s.io/node-api => k8s.io/node-api v0.19.0
```
Comment (pohly) [inline go.mod:26]: Why is node-api still v0.19.0?

Comment (humblec) [reply go.mod]: @pohly upgraded to 0.21.1 . Thanks! @pohly ptal.

**Review by pohly — APPROVED** [review]
/lgtm

**Pull request merged**